packages/label/src/LabelBase.tsx

```diff
@@ -0,0 +1,151 @@
+import {
+  critical20,
+  informative20,
+  success20,
+  warning20,
+} from '@littlespoon/theme/lib/colors/alert'
+import { brand60 } from '@littlespoon/theme/lib/colors/primary'
+import { gold60, limeGreen60, peach60, pink60 } from '@littlespoon/theme/lib/colors/secondary'
+import { shadeBlack } from '@littlespoon/theme/lib/colors/token'
+import { family, weight } from '@littlespoon/theme/lib/fonts/primary'
+import { rem } from '@littlespoon/theme/lib/utils'
+import styled from 'styled-components'
+
+import type { LabelProps } from './Label'
+
+export const LabelBase = styled.label<LabelProps<'label'>>`
+  align-items: center;
+  border-radius: 2px;
+  border: 0;
+  box-sizing: border-box;
+  display: inline-flex;
+  font: ${weight.bold} ${rem(2)} ${family};
+  letter-spacing: ${rem(0.1)};
+  text-decoration: none;
+  text-transform: uppercase;
+  width: fit-content;
+  line-height: 150%;
+  color: ${shadeBlack};
+  ${getSizeCss}
+  ${getVariantCss}
+`
+
+/**
+ * Gets size styles.
+ */
+function getSizeCss(props: LabelProps<'label'>): string {
+  const { size } = props
+  if (!size) {
+    return ''
+  }
+  let height = ''
+  let padding = ''
+  let fontSize = ''
+
+  switch (size) {
+    case 'small':
+      height = rem(2.2)
+      padding = `${rem(0.2)} ${rem(0.8)}`
+      fontSize = rem(1.2)
+      break
+
+    case 'medium':
+      height = rem(3.2)
+      padding = `${rem(0.4)} ${rem(1.6)}`
+      fontSize = rem(1.6)
+      break
+
+    case 'large':
+      height = rem(4.2)
+      padding = `${rem(0.6)} ${rem(2.4)};`
+      fontSize = rem(2)
+      break
+  }
+
+  return `
+    font-size: ${fontSize};
+    height: ${height};
+    padding: ${padding};
+  `
+}
+
+/**
+ * Gets variant styles.
+ */
+function getVariantCss(props: LabelProps<'label'>): string {
+  const { variant } = props
+
+  let backgroundColor = ''
+
+  /**
+   * {@link https://zeroheight.com/3ddd0f892/p/01a397-labels/t/190120}
+   */
+  switch (variant) {
```

Comment:
Nit: alternatively can use a object map instead of a switch statement, as the latter is just assigning a value:

```ts
const variantToColorMap = {
  success: success20(),
  warning: warning20(),
  critical: critical20(),
  informative: informative20(),
  BLW: pink60(),
  'most-popular': brand60(),
  'picky-eater-fave': peach60(),
  'seasonal': gold60(),
  'veggie-packed': limeGreen60(),
}

return `background-color: ${variantToColorMap[variant]};`
```

Reply:
Better not to have predefined colors here. 

https://github.com/little-spoon-dev/design-system/pull/568/files/4b23dd942694282a7350f5491e897707158fff8a#r940817559

Although maybe the colors for statuses could make sense since they are generic enough.

Reply:
No longer use variant.